server/api/controllers/auth.py

```
@@ -255,3 +276,153 @@ def login():
+
+def github_connect():
+    """Initiate GitHub OAuth flow for account linking"""
+    auth = request.headers.get("Authorization")
+    if not auth or not auth.startswith("Bearer "):
+        return jsonify({"error": "Authentication required"}), 401
+
+    token = auth.split(" ")[1]
+    try:
+        decoded = jwt.decode(token, JWT_SECRET, algorithms=["HS256"])
+        user_id = decoded["id"]
+    except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)